src/mpi/errhan/dynerrutil.c

```diff
@@ -51,7 +51,7 @@ static int  first_free_code  = 1;  /* code 0 is reserved */
 static const char empty_error_string[1] = { 0 };
 
 /* Forward reference */
-const char *MPIR_Err_get_dynerr_string( int code );
+static const char *MPIR_Err_get_dynerr_string( int code );
```

Comment:
Can you also add static to the actual definition below? Another commit to remove the `MPIR_Err` prefixes from all the static functions here would be nice to have, as well.

Reply:
Done. I added the extra commit too. Didn't build test though, so we should probably restart the tests just in case I messed up.